Data/ByteString/Lazy/Char8.hs

```diff
@@ -758,30 +758,6 @@ lines (Chunk c0 cs0) = loop0 c0 cs0
                 let c' = revChunks (B.unsafeTake n c : line)
                  in c' `seq` (c' : loop0 (B.unsafeDrop (n+1) c) cs)
 
-{-
-
-This function is too strict!  Consider,
-
-> prop_lazy =
-    (L.unpack . head . lazylines $ L.append (L.pack "a\nb\n") (error "failed"))
-  ==
-    "a"
```

Comment:
Could we possibly add this to the test suite?

Reply:
Can I ask someone else to pitch in?

Reply:
I am a bit reluctant to backport two commits to `bytestring-0.11` branch, when one can make do with one. Please just add 
```haskell
prop_lines_lazy = head (LC.lines (LC.append (LC.pack "a\nb\n") undefined)) == LC.pack "a"
```
to `Properties.hs` and mention `prop_lines_lazy` in `ll_tests`, that's it.
```
    , testProperty "lines_lazy"         prop_lines_lazy
```

Reply:
Done.